<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proxy_settings</class>
 <widget class="QDialog" name="proxy_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Configure proxies to access internet</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButtonNoProxy">
        <property name="text">
         <string>No proxy</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButtonManualProxy">
        <property name="text">
         <string>Manual proxy</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QTableWidget" name="tableWidgetPoxyTable">
        <column>
         <property name="text">
          <string>Host</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Port</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>User name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Password</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Proxy type</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButtonAddProxy">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButtonTorProxy">
        <property name="text">
         <string>Tor proxy</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QGroupBox" name="groupBoxTorProxy">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Control port</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditTorPassword"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxTorPort">
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>9151</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditTorHost">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButtonDeleteProxy">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonHelp">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
